runtime/spark/scripts/install.sh

```diff
@@ -46,6 +46,20 @@ if [ ! -d "${SPARK_HOME}" ]; then
   echo "export PATH=\$SPARK_HOME/bin:\$PATH" >> ${USER_HOME}/.bashrc
 fi
 
+# Install Jupyter and spylon-kernel for Spark
+if ! type jupyter >/dev/null 2>&1; then
+  echo "Install JupyterLab..."
+  pip3 install jupyterlab
```

Comment:
Is there any specific reason that we use pip3 specifically? Use "pip" not working?

Reply:
Change it to `pip`.